<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>이학민</author>
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <widget class="QGroupBox" name="groupBox_button">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>69</y>
     <width>100</width>
     <height>411</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(79, 92, 101);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_windows">
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::TabFocus</enum>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">padding : 20px; border-radius : 15px;
border-bottom-color: rgb(255, 255, 255);
background-color: rgb(26, 190, 185);

font: 11pt &quot;휴먼둥근헤드라인&quot;;
color: rgb(255, 255, 255);

</string>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="text">
       <string>창문</string>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_fan">
      <property name="styleSheet">
       <string notr="true">padding : 20px; border-radius : 15px;
background-color: rgb(26, 190, 185);


font: 11pt &quot;휴먼둥근헤드라인&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>팬</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_blind">
      <property name="styleSheet">
       <string notr="true">padding : 20px; border-radius : 15px;
background-color: rgb(26, 190, 185);



font: 8pt &quot;휴먼둥근헤드라인&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>블라인드</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_door">
      <property name="styleSheet">
       <string notr="true">padding : 20px; border-radius : 15px;
background-color: rgb(26, 190, 185);


font: 10pt &quot;휴먼둥근헤드라인&quot;;
color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string>현관문</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_light">
      <property name="styleSheet">
       <string notr="true">padding : 20px; border-radius : 15px;
background-color: rgb(26, 190, 185);


font: 11pt &quot;휴먼둥근헤드라인&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>전등</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_cctv">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>78</y>
     <width>431</width>
     <height>401</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="label_cctv">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>401</width>
      <height>321</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_blind">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>-2</y>
     <width>581</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 85, 127);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QSlider" name="horizontalSlider_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>421</width>
      <height>19</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>블라인드</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-2</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(26, 190, 185);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_fan">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>-2</y>
     <width>581</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(255, 170, 127);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AcadEref</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>팬</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_light">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>-2</y>
     <width>581</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QPushButton" name="pushButton_light_switch1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>전등1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_light_switch2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>전등2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AcadEref</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>전등</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_door">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>-2</y>
     <width>581</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(26, 190, 185);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding : 30px; border-radius : 30px;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AcadEref</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>집문</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_windows">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>-2</y>
     <width>581</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 127);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>511</width>
      <height>19</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_windows_on">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>on</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_windows_off">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>off</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AcadEref</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>창문</string>
    </property>
   </widget>
  </widget>
  <widget class="QTimeEdit" name="timeEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>699</x>
     <y>10</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Aharoni</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="showGroupSeparator" stdset="0">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>320</y>
     <width>66</width>
     <height>137</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>빗물</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="lcdNumber_rain"/>
    </item>
    <item>
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>습도</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="lcdNumber_humidity"/>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>온도</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="lcdNumber_temperature"/>
    </item>
   </layout>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_api_temperature">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>90</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="smallDecimalPoint">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>110</y>
     <width>31</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>(℃)</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_api_humidity">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>160</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>160</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_api_pm25">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>200</y>
     <width>81</width>
     <height>64</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>220</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>PM2.5</string>
   </property>
  </widget>
  <zorder>groupBox_button</zorder>
  <zorder>groupBox_light</zorder>
  <zorder>groupBox_windows</zorder>
  <zorder>groupBox_blind</zorder>
  <zorder>groupBox_fan</zorder>
  <zorder>groupBox_door</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>groupBox_cctv</zorder>
  <zorder>frame</zorder>
  <zorder>timeEdit</zorder>
  <zorder>lcdNumber_api_temperature</zorder>
  <zorder>label_6</zorder>
  <zorder>lcdNumber_api_humidity</zorder>
  <zorder>label_7</zorder>
  <zorder>lcdNumber_api_pm25</zorder>
  <zorder>label_8</zorder>
 </widget>
 <layoutdefault spacing="6" margin="9"/>
 <resources>

 </resources>
 <connections>
  <connection>
   <sender>pushButton_windows</sender>
   <signal>clicked(bool)</signal>
   <receiver>groupBox_windows</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>438</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_windows</sender>
   <signal>clicked(bool)</signal>
   <receiver>groupBox_fan</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>25</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_windows</sender>
   <signal>clicked(bool)</signal>
   <receiver>groupBox_blind</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>18</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_windows</sender>
   <signal>clicked(bool)</signal>
   <receiver>groupBox_door</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_windows</sender>
   <signal>clicked(bool)</signal>
   <receiver>groupBox_light</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>452</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_fan</sender>
   <signal>clicked(bool)</signal>
   <receiver>groupBox_fan</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_fan</sender>
   <signal>clicked(bool)</signal>
   <receiver>groupBox_windows</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>680</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_fan</sender>
   <signal>clicked(bool)</signal>
   <receiver>groupBox_blind</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_fan</sender>
   <signal>clicked(bool)</signal>
   <receiver>groupBox_door</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_fan</sender>
   <signal>clicked(bool)</signal>
   <receiver>groupBox_light</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_blind</sender>
   <signal>clicked(bool)</signal>
   <receiver>groupBox_blind</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_blind</sender>
   <signal>clicked(bool)</signal>
   <receiver>groupBox_windows</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>480</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_blind</sender>
   <signal>clicked(bool)</signal>
   <receiver>groupBox_fan</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_blind</sender>
   <signal>clicked(bool)</signal>
   <receiver>groupBox_door</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>29</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_blind</sender>
   <signal>clicked(bool)</signal>
   <receiver>groupBox_light</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>23</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_door</sender>
   <signal>clicked(bool)</signal>
   <receiver>groupBox_door</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_door</sender>
   <signal>clicked(bool)</signal>
   <receiver>groupBox_light</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_door</sender>
   <signal>clicked(bool)</signal>
   <receiver>groupBox_blind</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_door</sender>
   <signal>clicked(bool)</signal>
   <receiver>groupBox_fan</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_door</sender>
   <signal>clicked(bool)</signal>
   <receiver>groupBox_windows</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel">
     <x>498</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_light</sender>
   <signal>clicked(bool)</signal>
   <receiver>groupBox_light</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>452</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_light</sender>
   <signal>clicked(bool)</signal>
   <receiver>groupBox_door</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>452</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_light</sender>
   <signal>clicked(bool)</signal>
   <receiver>groupBox_blind</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>450</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_light</sender>
   <signal>clicked(bool)</signal>
   <receiver>groupBox_fan</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>452</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_light</sender>
   <signal>clicked(bool)</signal>
   <receiver>groupBox_windows</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>36</x>
     <y>452</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
